src/components/ButtonPanel.js

```diff
@@ -1,12 +1,9 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { v4 as genKey } from 'uuid';
 import Button from './Button';
 
-const renderGroup = group => (
-  group.map(item => <Button name={item} key={genKey()} wide={item === '0'} />)
-);
-
-const ButtonPanel = () => {
+const ButtonPanel = props => {
```

Comment:
[Optional]
You could destructure clickhandler at the argument level `const ButtonPanel = ({clickhandler}) => {`